.github/workflows/unit-tests.yml

```diff
@@ -1,27 +1,18 @@
-name: Run unit tests
+name: CI
 
-on:
-  - push
-  - pull_request
+on: pull_request
 
-jobs:
-  test:
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
 
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ruby: ["2.7"]
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
 
-    steps:
-    - uses: actions/checkout@v2
-    - name: Add hammer-cli-foreman to local gem file
-      run: echo "gem 'hammer_cli_foreman', :git => 'https://github.com/theforeman/hammer-cli-foreman.git'" > Gemfile.local
-    - name: Add hammer-cli to local gem file
-      run: echo "gem 'hammer_cli', :git => 'https://github.com/theforeman/hammer-cli.git'" >> Gemfile.local
```

Comment:
These are now only pulled in via released gems, no longer the git version. If you care about this, I'd recommend adding it to your Gemfile. If you want, you can limit it to `ENV['GITHUB_ACTIONS'] == 'true'`.